code/src/components/Styles.scss

```diff
@@ -0,0 +1,185 @@
+/*** VARIABLES ***/
+
+$gray: #f2f0f0;
+$black: #000;
+$pink: #f4acac;
+$light-pink: #ffd5cd;
+$red: red;
+$green: green;
+
+/*** GENERAL ***/
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  font-family: menlo;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Is this necessary? I understand that it supposed to help reducing jagged edges and noticeably improves image quality - but we don't really have images that would benefit from this?